Resources/KwallyKwik.rdef

```diff
@@ -1954,9 +1954,9 @@ resource(5, "InexactMatchWindow") #'ARCV' archive BWindow {
 				"_color" = -656877568,
 				"_ploc" = point { 2.00000, 50.0000 },
 				#'CSTR' array "_text" =  array {
```

Comment:
Have you checked if the proper fix is just to uncomment this line?

Reply:
This issue references this problem...
https://github.com/HaikuArchives/UltraEncode/issues/3
The issue seems to put a comment on line 1956, so I assumed the rest of the statement should be commented, too.
But I'm not very sure. This is just an assumption :(

Reply:
Yes, please try uncommenting this line.